### PR TITLE
[one-cmds] Introduce make_cmd.py

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit introduces make_cmd.py to one-cmds.

Related: https://github.com/Samsung/ONE/issues/8169
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/one-cmds/one-import-bcq [hunk 1]
```diff
@@ -25,6 +25,7 @@ import subprocess
 import sys
 import tempfile
 
+import python.make_cmd as _make_cmd
```
Comment (seanshpark) [inline compiler/one-cmds/one-import-bcq:28]: I think `python.make_cmd` came from using folder name `python`.
Which looks a bit strange as the this is implemented in python.
What I could think of is like `onelib` or `cmdslib`; not using generic word that can confuse code readers.
What do you think?

Comment (mhs4670go) [reply compiler/one-cmds/one-import-bcq:28]: @seanshpark Good idea. It'd really confuse people. How about merge this and change all at once?

Comment (seanshpark) [reply compiler/one-cmds/one-import-bcq:28]: OK

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**